Otherwise building for riscv64 would end up with the following error in the log:

```
.. Building cross-gcc (compiler)...
!! Build failed!
!. Last lines of the log file (/mussel/log.txt):
checking for fcntl.h... (cached) yes
checking whether O_NONBLOCK is declared... yes
checking for personality ADDR_NO_RANDOMIZE... yes
checking for AF_UNIX... yes
checking for AF_INET6... yes
checking for _LK_LOCK... no
checking if mkdir takes one argument... no
This target does not support --with-cpu.
Valid --with options are: abi arch tune riscv_attribute isa_spec tls cmodel
make: *** [Makefile:4670: configure-gcc] Error 1

```

--- a/mussel
+++ b/mussel
@@ -442,7 +442,7 @@ while [ $# -gt 0 ]; do
       XARCH=riscv64
       LARCH=riscv
       MARCH=$XARCH
-      XGCCARGS="--with-mcpu=sifive-u74 --with-arch=rv64gc --with-tune=sifive-7-series --with-abi=lp64d"
+      XGCCARGS="--with-arch=rv64gc --with-tune=sifive-7-series --with-abi=lp64d"
       XPURE64=$XARCH
       XTARGET=$XARCH-linux-musl
       ;;

--- a/mussel
+++ b/mussel
@@ -442,7 +442,7 @@ while [ $# -gt 0 ]; do
       XARCH=riscv64
       LARCH=riscv
       MARCH=$XARCH
-      XGCCARGS="--with-arch=rv64gc --with-tune=sifive-7-series --with-abi=lp64d"
+      XGCCARGS="--with-mcpu=sifive-u74 --with-arch=rv64gc --with-tune=sifive-7-series --with-abi=lp64d"
       XPURE64=$XARCH
       XTARGET=$XARCH-linux-musl
       ;;

